internal/presenters/templates/local_finding.tmpl

```diff
@@ -1,15 +1,15 @@
 {{- define "details" }}
 {{- if gt ($.OpenFindings | len) 0 }}{{ "Open Issues" | title }}
```

Comment:
**issue:** Remove usage of injected `OpenFindings` and `IgnoredFindings`, instead process LocalFindings struct directly.